games-arcade/blobbyvolley2/blobbyvolley2-1.1~git.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Official continuation of the famous Blobby Volley 1.x arcade game"
+DESCRIPTION="The head-to-head multiplayer ball game
+
+The \"Dedicated Server\" runs with a Gamespeed of 100%, which means 75 FPS
+The Port for the Server is 1234."
+HOMEPAGE="https://github.com/danielknobe/blobbyvolley2"
+COPYRIGHT="2006 Daniel Knobe
```

Comment:
Maybe?
```diff
- COPYRIGHT="2006 Daniel Knobe
+ COPYRIGHT="2006-2020 Daniel Knobe
```

Reply:
Yeah, I'm not sure when to update those so I just copied it from the files. The thing is, Jonathan Sieber probably contributed after 2006, but I can't pinpoint their commits.

Reply:
I see lots of commits to the source from https://github.com/ngc92 up to 2015, not sure who's name is related to that? Maybe @danielknobe can shed a light here?

Reply:
ngc92 is Erik Schultheis.
yonibear is Jonathan Sieber.

Reply:
Okay, I listed all the programmers in authors now, although I'm not sure if I'm supposed to add those who don't have existing copyright statements.